MediaProcessor/cmake/src.cmake

```diff
@@ -28,21 +28,39 @@ if(APPLE)
     endif()
 elseif(WIN32)
     set(DF_LIBRARY ${CMAKE_SOURCE_DIR}/lib/libdf.dll.a) # .dll.a for linking, .dll for runtime
+    set(DF_DLL_PATH ${CMAKE_SOURCE_DIR}/lib/df.dll)   # Path for the actual DLL
```

Comment:
Why was this needed? Normally providing the .dll.a is enough for linking?

Reply:
It was needed by the command to move the dll to the app's directory. 
Because On windows the mediaprocess.exe doesn't give expected output. So I used dependency walker to check that and find out that the mediaprocessor.exe is unable to find runtime lib.

should i move the the function here?

Reply:
Okay makes sense. I probably had the dll on my path and didn't notice; good catch